coderefs/coderefs.go

```diff
@@ -16,20 +14,15 @@ import (
 	"github.com/launchdarkly/ld-find-code-refs/search"
 )
 
-const (
-	minFlagKeyLen    = 3  // Minimum flag key length helps reduce the number of false positives
-	maxProjKeyLength = 20 // Maximum project key length
-)
-
-// Scan checks the configured directory for flags base on the options configured for Code References.
-func Scan(opts options.Options) {
+func Run(opts options.Options) {
 	dir := opts.Dir
 	absPath, err := validation.NormalizeAndValidatePath(dir)
 	if err != nil {
 		log.Error.Fatalf("could not validate directory option: %s", err)
 	}
 
 	log.Info.Printf("absolute directory path: %s", absPath)
+	ldApi := ld.InitApiClient(ld.ApiOptions{ApiKey: opts.AccessToken, BaseUri: opts.BaseUri, ProjKey: opts.ProjKey, UserAgent: "LDFindCodeRefs/" + version.Version})
```

Comment:
It seems `InitApiClient` returns a new `ApiClient` object every time. We call this init function again in `flags.go` [here](https://github.com/launchdarkly/ld-find-code-refs/blob/05b2b82f6253169a829d844c75a951148480dbec/flags/flags.go#L23) when doing the scan.  I don't think this is a big deal, but there's a chance we can re-use this first object and avoid having to re-initialise the client, which might give us a little perf boost.

Reply:
Same as above. I'll note this down for a follow up to investigate in a smaller PR. It's how it works today so for now leaving as is.